File: tests/expected/built_ins.kt
import kotlin.math.max
import kotlin.math.min

fun default_builtins() {
    val a = ""
    val b = false
    val c = 0
    val d = 0.0
    assert(a == "")
    assert(b == false)
    assert(c == 0)
    assert(d == 0.0)
}

fun main(argv: Array<String>) {
    val a = max(1, 2)
    println("$a")
    val b = min(1, 2)
    println("$b")
    val c = min(1.0, 2.0).toInt()
    println("$c")
}

File: tests/expected/dart/built_ins.dart
// @dart=2.9
import 'dart:math';
import 'package:sprintf/sprintf.dart';

default_builtins() {
  final String a = "";
  final bool b = false;
  final int c = 0;
  assert(a == "");
  assert(b == false);
  assert(c == 0);
}

main(List<String> argv) {
  final int a = max(1, 2);
  print(sprintf("%s", [a]));
  final int b = min(1, 2);
  print(sprintf("%s", [b]));
}
